src/EA.Weee.Web/Areas/AATFReturn/Controllers/SelectReportOptionsController.cs

```diff
@@ -71,7 +72,7 @@ public virtual async Task<ActionResult> Index(SelectReportOptionsViewModel viewM
 
             if (ModelState.IsValid)
             {
-                if (viewModel.SelectedOptions != null)
+                if (viewModel.SelectedOptions != null && viewModel.SelectedOptions.Count != 0)
```

Comment:
Can add a property on the view model HasSelectedOptions that would do the null and count checks